docs/README.md

```diff
@@ -1,58 +1,44 @@
-# VTEX React App Template
+# VTEX Product List Context app
 
-Our guide repository to structure for react apps, that should be used as a template.
+This app is the Product List Context that should be used to store certain products. Example of usage: This is used on the `Shelf` app to store the visible products.
```

Comment:
The Product List Context is an app responsible for fetch product data and then provide it to some store product blocks, such as the Shelf.

:warning: This app is not rendered and should only be used to provide product data to specific store blocks.